site/plugins.json

```diff
@@ -477,24 +477,26 @@
     "name": "Essential Next.js",
     "package": "@netlify/plugin-nextjs",
     "repo": "https://github.com/netlify/netlify-plugin-nextjs",
-    "version": "4.2.2",
+    "version": "4.2.3",
     "compatibility": [
       {
-        "version": "4.2.2",
+        "version": "4.2.3",
         "migrationGuide": "https://ntl.fyi/next-plugin-migration"
       },
-      {
-        "version": "3.9.2",
-        "siteDependencies": {
-          "next": "<10.0.9"
+      [
+        {
+          "version": "3.9.2",
+          "siteDependencies": {
+            "next": "<10.0.9"
+          }
+        },
+        {
+          "version": "1.1.5",
+          "siteDependencies": {
+            "next": "<10.0.6"
+          }
         }
-      },
-      {
-        "version": "1.1.5",
-        "siteDependencies": {
-          "next": "<10.0.6"
-        }
-      }
+      ]
```

Comment:
Tests are failing because this is invalid syntax. `compatibility` should be an array of objects, not an array of arrays of objects. The code injecting might be missing some `Array.flat()`.